app/components/add-contributor/component.js

```diff
@@ -63,12 +62,13 @@ export default Component.extend({
         this.reset();
         window.scroll(0,0);
```

Comment:
Same as above, the reset and scrolling is not necessary. I guess that's still from when the form was on the index page.